runtime/vm/mz64/UpcallThunkGen.cpp

```diff
@@ -287,6 +287,7 @@ createUpcallThunk(J9UpcallMetaData *metaData)
 	}
 	for (I_32 i = 0; i < lastSigIdx; i++) {
 		U_32 argSizeInBytes = sigArray[i].sizeInByte;
+		const I_32 totalStackSlotsReq = (argSizeInBytes + 7) / 8;
```

Comment:
Suggest the type of `currStackSlotIndex` and `totalStackSlotsReq` should be `U_32` to avoid mixing signed and unsigned types.

Reply:
Should be fixed now: https://github.com/eclipse-openj9/openj9/commit/fc65fff69564999a4fe6a34247e6cd6dcb191aab